src/sqlfluff/cli/autocomplete.py

```diff
@@ -1,9 +1,15 @@
 """autocompletion commands."""
 
-from click.shell_completion import CompletionItem
-
 from sqlfluff import list_dialects
 
+# Older versions of click don't have shell completion
+# so handle for now, as version 8 still fairly recent
+shell_completion_enabled = True
+try:
+    completion = __import__("click.shell_completion", fromlist=["click"])
```

Comment:
You can use the normal.`import` command here -- no need to use the function.

Reply:
Oh yeah. Went round the houses as this was originally importing just one class where that doesn't work when the module doesn't exist. But works fine now just importing the module.

Fixed!